src/main/java/uk/gov/hmcts/darts/common/entity/DailyList.java

```diff
@@ -10,63 +10,96 @@
 @Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class DailyList {
 
+    public static final String ID = "moj_dal_id";
+    public static final String COURTHOUSE_ID = "moj_crt_id";
+    public static final String DAILY_LIST_OBJECT_ID = "r_daily_list_object_id";
+    public static final String UNIQUE_ID = "c_unique_id";
+    public static final String JOB_STATUS = "c_job_status";
+    public static final String PUBLISHED_TIME = "c_timestamp";
+    public static final String DAILY_LIST_ID = "c_daily_list_id";
+    public static final String START_DATE = "c_start_date";
+    public static final String END_DATE = "c_end_date";
+    public static final String DAILY_LIST_ID_STRING = "c_daily_list_id_s";
+    public static final String DAILY_LIST_SOURCE = "c_daily_list_source";
+    public static final String CREATED_DATE_TIME = "created_ts";
+    public static final String LAST_UPDATED_DATE_TIME = "last_modified_ts";
+    public static final String LEGACY_COURTHOUSE_OBJECT_ID = "r_courthouse_object_id";
+    public static final String LEGACY_VERSION_LABEL = "r_version_label";
+    public static final String SUPERSEDED = "i_superseded";
+    public static final String VERSION_LABEL = "i_version_label";
+    public static final String TABLE_NAME = "moj_daily_list";
+
     @Id
-    @Column(name = "moj_dal_id")
+    @Column(name = ID)
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_dal_gen")
     @SequenceGenerator(name = "moj_dal_gen", sequenceName = "moj_dal_seq", allocationSize = 1)
     private Integer id;
 
     @ManyToOne
-    @JoinColumn(name = "moj_crt_id")
-    private Courthouse theCourthouse;
+    @JoinColumn(name = COURTHOUSE_ID)
+    private Courthouse crtId;
 
-    @Column(name = "r_daily_list_object_id", length = 16)
-    private String legacyObjectId;
+    @Column(name = DAILY_LIST_OBJECT_ID)
+    private String dailyListObjectId;
 
-    @Column(name = "c_unique_id")
-    private String uniqueId;
+    @Column(name = UNIQUE_ID)
+    private String uniqueID;
 
-    @Column(name = "c_job_status")
+    @Column(name = JOB_STATUS)
     private String jobStatus;
 
-    @Column(name = "c_timestamp")
-    private OffsetDateTime timestamp;
+    @Column(name = PUBLISHED_TIME)
+    private OffsetDateTime publishedTime;
 
-    @Column(name = "c_daily_list_id")
-    private Integer dailyListId;
+    @Column(name = DAILY_LIST_ID)
+    private Integer dailylistId;
 
-    @Column(name = "c_start_date")
+    @Column(name = START_DATE)
     private OffsetDateTime startDate;
 
-    @Column(name = "c_end_date")
+    @Column(name = END_DATE)
     private OffsetDateTime endDate;
 
-    @Column(name = "c_daily_list_id_s", length = 100)
+    @CreationTimestamp
+    @Column(name = CREATED_DATE_TIME)
+    private OffsetDateTime createdDateTime;
+
+    @UpdateTimestamp
+    @Column(name = LAST_UPDATED_DATE_TIME)
+    private OffsetDateTime lastUpdatedDateTime;
+
+    @Column(name = DAILY_LIST_ID_STRING, length = 100)
```

Comment:
Please can we remove the `length` property from the `CHARACTER VARYING` columns to keep inline with Barry's v21.sql?